### DISCUSSION TITLE
how to generate determined result

### DISCUSSION THREAD

**Discussion by DeoLeung** [SYSTRAN/faster-whisper#880] [Q&A]
I'm adding faster whisper to my test ci, using the smallest model inferencing on cpu

how can I enforce it to generate a determined result?

**Comment by trungkienbkhn** [comment]
@DeoLeung , hello. you can refer to this [eample](https://github.com/SYSTRAN/faster-whisper/blob/master/tests/test_transcribe.py#L32).

**Reply by DeoLeung** [reply]
I can't see any params to make output "determined" :(

usually we have ci server using linux, and various dev machine like mac m1/x86, using the `tiny` model on Chineses outputs slight different things, even on identical machine.

currently we skip testing the output text, just test the output structure to ensure it works.
